test/test_cli_integration.py

```diff
@@ -31,6 +31,8 @@ class CliIntegrationTest(unittest.TestCase): # pragma: no cover
     HTTPS_CA_BUNDLE_ENV_VAR = { 'CONJUR_CA_BUNDLE': 'ca-bundle' }
 
     DEFINED_VARIABLE_ID = 'one/password'
+    DEFINED_VARIABLE2 = 'two/username'
+    DEFINED_VARIABLE3 = 'three/secret'
```

Comment:
We should indicate that they are variable IDs like the initial L33 one:
```suggestion
    DEFINED_VARIABLE_ID_2 = 'two/username'
    DEFINED_VARIABLE_ID_3 = 'three/secret'
```